deployments/helm/onos-gui/templates/deployment.yaml

```diff
@@ -0,0 +1,117 @@
+    {{- end }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          stdin : true
+          volumeMounts:
+            - name: config
+              mountPath: /home/onos/.atomix/config.yaml
+              subPath:   atomix.yaml
+              readOnly: false
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          ports:
+            {{- with .Values.ports }}
+            {{- range $key, $port := . }}
+            - name: {{ $key }}
+              {{ toYaml $port | nindent 14 }}
+            {{- end }}
+            {{- end }}
+          livenessProbe:
+            tcpSocket:
+              port: web
+            initialDelaySeconds: 30
+          readinessProbe:
+            tcpSocket:
+              port: web
+            initialDelaySeconds: 30
+        - name: onos-envoy
+          image: "{{ .Values.Envoy.image.repository }}:{{ .Values.Envoy.image.tag }}"
+          imagePullPolicy: {{ .Values.Envoy.image.pullPolicy }}
+          command:
+            {{ toYaml .Values.Envoy.command | nindent 12 }}
+          args:
+            {{- if $.Values.Envoy.argsTemplate }}
+              {{ tpl $.Values.Envoy.argsTemplate $ | nindent 12}}
+              {{- else }}
+              {{ toYaml .Values.Envoy.args | nindent 12 }}
+              {{- end }}
+          ports:
+            {{- with .Values.Envoy.ports }}
+            {{- range $key, $port := . }}
+            - name: {{ $key }}
+              {{ toYaml $port | nindent 14 }}
+            {{- end }}
+            {{- end }}
+            {{- range $key, $value := .Values.onosservices }}
+            - name: {{ $key }}-port
+              containerPort: {{ $value.proxy }}
+              protocol: TCP
+            {{- end }}
+
+          livenessProbe:
+            {{ toYaml .Values.Envoy.livenessProbe | nindent 12 }}
+          readinessProbe:
+            {{ toYaml .Values.Envoy.readinessProbe | nindent 12 }}
+          env:
+          {{- range $key, $value := .Values.Envoy.env }}
+          - name: {{ $key | upper | replace "." "_" }}
+            value: {{ $value | quote }}
+          {{- end }}
+          resources:
+            {{ toYaml .Values.Envoy.resources | nindent 12 }}
+          volumeMounts:
+            - name: config
+              mountPath: /config
+            - name: secret
+              mountPath: /etc/envoy-proxy/certs
+              readOnly: true
+            {{- if .Values.Envoy.volumeMounts }}
+            {{ toYaml .Values.Envoy.volumeMounts | nindent 12 }}
+            {{- end }}
+          lifecycle:
+            {{ toYaml .Values.Envoy.lifecyle | nindent 12 }}
+
+      volumes:
+        - name: config
```

Comment:
I believe this is used just in atomix value you referent prior and I'm quite sure you do not need it.

Reply:
no I think it is necessary - I need to have a configMap - when the envoy.yaml is expanded out it is made available in this config map at `/config/envoy.yaml`